packages/relay-kit/src/packs/safe-4337/Safe4337Pack.ts

```diff
@@ -111,6 +115,15 @@ export class Safe4337Pack extends RelayKitBasePack<{
     this.#ENTRYPOINT_ADDRESS = entryPointAddress
     this.#SAFE_4337_MODULE_ADDRESS = safe4337ModuleAddress
     this.#SAFE_WEBAUTHN_SHARED_SIGNER_ADDRESS = safeWebAuthnSharedSignerAddress || '0x'
+
+    if (onchainAnalitics?.project) {
+      this.#onchainIdentifier = generateOnChainIdentifier(
+        onchainAnalitics.project,
+        onchainAnalitics.platform,
+        'relay-kit',
+        '3.2.4'
```

Comment:
Hardcoded version ? Should be coming from package.json

Reply:
done! I added a `getProtocolKitVersion` and a `getRelayKitVersion` to automatically get the package version  from the `package.json`